lesson2.tex

```diff
@@ -46,56 +45,111 @@
   \section{Procedure}
   
     \subsection{Setup (10 minutes)}
-      Turn on computers and open the programming environment.
+      Turn on computers and open the programming environment. Construct a tape obstacle course on a table that students are able to navigate through. Ideally, there should be an easy course and a hard course. Two examples of these courses are shown in the appendix. 
```

Comment:
No appendix was included. Show a diagram (preferably written in Latex) with the obstacle courses. It should require students to use all the skills learned in this lesson (move forward, backward, turn). Also, this should be included here, not in an appendix.